docs/instance-customisation/what-can-i-customise.md

```diff
@@ -0,0 +1,42 @@
+## What can I customise?
+The Instance Customisation Plugin API exposes a range of configurable options for your Scalable Pixel Streaming applications. Scalable Pixel Streaming will send what is currently configured as part of your Application to your instance customisation plugin and you have the option to modify these options as you see fit.
+
+### Plugin Options
+One of the key features to instance customisation is the ability to send data from your frontend implementation through to your instance customisation plugin at the time of requesting an instance.
+
+This is a string that can be formatted to send any number of arbitrary options from your frontend to your instance customisation plugin and it might look something like the following:
+
+```json
+{
+   "user": "UserA",
+   "map": "Level_1",
+   "region": ""
+}
+```
+
+Keeping in mind that this data can represent **anything that you want**. It’s up to you what you do with this in your instance customisation plugin.
+
+### Runtime Options
+Runtime Options provides your instance customisation plugin with access to the configured options from your instances Application. These will be sent back to Scalable Pixel Streaming when requesting an instance, so any modifications your instance customisation plugin makes to these options will be reflected.
```

Comment:
Change:

> Runtime Options provides your instance customisation plugin with access to the configured options from your instances Application.

to:

> Runtime Options provide your instance customisation plugin access to the configured options from your instance's Application.

I think this logic checks out here for instances (aka the instance's application). Feels a little odd to have an ownership by a child process so let me revisit in the morning.


I know its obvious, but maybe change to: reflected at runtime

Reply:
Is this what you meant?

> Runtime Options provide your instance customisation plugin access to the configured options reflected at runtime.

or...

> Runtime Options provide your instance customisation plugin access to the configured options from your instance's Application reflected at runtime.

Reply:
Sorry you're right this wasn't clear enough. Lets make it:

> Runtime Options allow your instance customisation plugin to access the configurations you have set for a given Version (e.g. resolution). These configurations will be sent back to Scalable Pixel Streaming when requesting an instance, so any modifications your instance customisation plugin makes to these options will be reflected at runtime (e.g. where instance 1 has a resolution of 1920x1080 and instance 2 has a resolution of 1280x720).